docs/tutorials/webxr-360-image.md

```diff
@@ -9,7 +9,7 @@ import Link from '@docusaurus/Link';
 360 Image with WebXR support | Image Credit: Bob Dass from https://www.flickr.com/photos/54144402@N03/
 
 <div className="iframe-container">
-    <iframe loading="lazy" src="https://playcanv.as/p/v6qoi4Yt/" title="WebXR 360 Image" webkitallowfullscreen="true" mozallowfullscreen="true" allow="autoplay" allowfullscreen="true" allowvr="" scrolling="no" frameborder="0" />
+    <iframe loading="lazy" src="https://playcanv.as/p/v6qoi4Yt/" title="WebXR 360 Image" webkitallowfullscreen="true" mozallowfullscreen="true" allow="autoplay *;xr-spatial-tracking *" allowfullscreen="true" allowvr="" scrolling="no" frameborder="0" />
```

Comment:
What are the asterisks for? See: https://www.radiantmediaplayer.com/blog/iframe-autoplay-policy.html example.

Reply:
Weird, when was testing it didn't like it without *, but now tested, and you are correct - it works without *.